index.md

```diff
@@ -2,8 +2,8 @@
 layout: frontpage
 
 # Header texts
-headerTitle: "Object-Oriented Meets Functional"
-headerSubtitle: "Have the best of both worlds. Construct elegant class hierarchies for maximum code reuse and extensibility, implement their behavior using higher-order functions. Or anything in-between."
+headerTitle: "The Scala Programming Language"
+headerSubtitle: "Scala is a concise high-level programming language. Scala's compiler helps you avoid bugs when writing complex applications, its fast runtime lets you build efficient, high-performance systems, with easy access to a huge ecosystem of libraries."
```

Comment:
I think you forgot an "and" before "its fast runtime".

Reply:
@gmethvin good catch, there is a missing `and` somewhere, but even with it added before `its fast runtime` sounds a bit odd. May be better to split it into two separate sentences

Reply:
nm I just added the `and` there and it seems to read fine